.github/actions/install-dependencies/action.yml

```
@@ -15,11 +15,18 @@ runs:
         sudo apt-get install -y \
           librsvg2-bin \
           ghostscript \
-          graphviz \
           pkg-config \
-          libgraphviz-dev \
           gsfonts \
           libfreetype6-dev \
           libfontconfig1-dev
       # Explanation of some of these dependencies:
       # https://github.com/ImageMagick/ImageMagick/issues/374#issuecomment-279252866
+
+    - uses: conda-incubator/setup-miniconda@35d1405e78aa3f784fe3ce9a2eb378d5eeb62169 # v2.1.1
+      with:
+        channels: conda-forge
+        python-version: ${{ matrix.python-version || '' }}
```

I think we should switch to use the open miniforge or mambaforge offerings rather than the propitiatory miniconda.

From [the docs](https://github.com/conda-incubator/setup-miniconda#example-10-miniforge) it looks like this will do it.

```suggestion
        python-version: ${{ matrix.python-version || '' }}
        miniforge-version: latest
```